EIPS/eip-1193.md

```diff
@@ -27,16 +27,19 @@ The events `connect`, `disconnect`, `chainChanged`, and `accountsChanged` are pr
 Makes an Ethereum RPC method call.
 
 ```typescript
-type RequestParams = Array<any> | { [key: string]: any };
+interface RequestArguments {
+  method: string;
+  params?: Array<any> | { [key: string]: any };
```

Comment:
This part confuses me because it seems to indicate that the `params` property can either be an array of stuff or an object. But the JSON-RPC only ever lists them out as arrays.